fastplotlib/graphics/linecollection.py

```diff
@@ -2,10 +2,9 @@
 import pygfx
 from typing import Union, List
 
-from .line import LineGraphic
+from fastplotlib.graphics.line import LineGraphic
```

Comment:
always use relative imports from within a package, I'm guessing this was pycharm's fault

Reply:
it likes to do this when doing the automated refactoring :weary: